src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/UpToDate/BuildUpToDateCheck.cs

```diff
@@ -823,11 +823,11 @@ async Task<(bool, ImmutableArray<ProjectConfiguration>)> CheckAsync(UpToDateChec
                             configurationLastCheckedAtUtc = DateTime.MinValue;
                         }
 
-                        if (!CheckGlobalConditions(logger, configurationLastCheckedAtUtc, validateFirstRun: !isValidationRun, implicitState) ||
-                            !CheckInputsAndOutputs(logger, configurationLastCheckedAtUtc, timestampCache, implicitState, ignoreKinds, token) ||
-                            !CheckMarkers(logger, timestampCache, implicitState) ||
-                            !CheckCopyToOutputDirectoryFiles(logger, timestampCache, implicitState, token) ||
-                            !CheckCopiedOutputFiles(logger, timestampCache, implicitState, token))
+                        if (!CheckGlobalConditions(logger, configurationLastCheckedAtUtc, validateFirstRun: !isValidationRun, implicitState)
+                            || !CheckInputsAndOutputs(logger, configurationLastCheckedAtUtc, timestampCache, implicitState, ignoreKinds, token)
+                            || !CheckMarkers(logger, timestampCache, implicitState)
+                            || !CheckCopyToOutputDirectoryFiles(logger, timestampCache, implicitState, token)
+                            || !CheckCopiedOutputFiles(logger, timestampCache, implicitState, token))
```

Comment:
You didn't have to change this! 😝 It was just a comment about it. If we want this style to be standard, we should just add a rule/analyzer in our editorconfig about it. I'm sure there is one that would match this style.

Reply:
Sounds like we should decide if we want that to be the standard then.